CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+4.0.0
+=====
+
+- Successful login now always return a `expiry` field for when the token expires.
```

Comment:
I would mention the breaking change and action to be taken in order to upgrade.

Reply:
@johnraz do you think I need to mention more? afaik unless you rely on the create() method directly, the upgrade path should be quite simple no?

Reply:
Yes I think that anybody could be using the AuthToken model with custom view to handle various things (like the refresh-token view we are missing).